examples/unstructured_profilers.ipynb

```diff
@@ -0,0 +1,387 @@
+    "        \"times\": dict(float)\n",
+    "    }\n",
+    "}\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5fcb5447",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "import json\n",
+    "sys.path.insert(0, '..')\n",
+    "import dataprofiler as dp\n",
+    "\n",
+    "data_path = \"../dataprofiler/tests/data\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a7fc2df6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = dp.Data(os.path.join(data_path, \"txt/discussion_reddit.txt\"))\n",
+    "profile = dp.Profiler(data)\n",
+    "\n",
+    "report  = profile.report(report_options={\"output_format\": \"pretty\"})\n",
+    "print(json.dumps(report, indent=4))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4d183992",
+   "metadata": {},
+   "source": [
+    "## Profiler Type"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d7ec39d2",
+   "metadata": {},
+   "source": [
+    "It should be noted, in addition to reading the input data from text files, DataProfiler allows the input data as a pandas dataframe, a pandas series, and a list if the Profiler is explicitly chosen as unstructured."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "29737f25",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# run data profiler and get the report\n",
+    "import pandas as pd\n",
+    "my_dataframe = pd.DataFrame([[\"Sample1\"],[\"Sample2\"],[\"Sample3\"]], columns=[\"Text_Samples\"])\n",
+    "profile = dp.Profiler(my_dataframe, profiler_type='unstructured')\n",
+    "\n",
+    "report  = profile.report(report_options={\"output_format\":\"pretty\"})\n",
+    "print(json.dumps(report, indent=4))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fe02ad64",
+   "metadata": {},
+   "source": [
+    "## Profiler options"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "40804cc9",
+   "metadata": {},
+   "source": [
+    "The DataProfiler has the ability to turn on and off components as needed. This is accomplished via the `ProfilerOptions` class.\n",
+    "\n",
+    "For example, if a user doesn't require vocab count information they may desire to turn off the word count functionality.\n",
+    "\n",
+    "Below, let's remove the vocab count and set the stop words. \n",
+    "\n",
+    "Full list of options in the Profiler section of the [DataProfiler documentation](https://capitalone.github.io/DataProfiler)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9d25d899",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = dp.Data(os.path.join(data_path, \"txt/discussion_reddit.txt\"))\n",
+    "\n",
+    "profile_options = dp.ProfilerOptions()\n",
+    "\n",
+    "# Set multiple options at the same time via set\n",
+    "profile_options.set({ \"vocab.is_enabled\": False })\n",
```

Comment:
You mention setting multiple options but just set 1, maybe we set 2 or three on this line? I believe the same thing occurs in the structured profiler notebook as well where it only sets 1 despite the comment alluding to setting multiple